pallets/identity/src/lib.rs

```diff
@@ -2124,6 +2124,47 @@ impl<T: Trait> Module<T> {
 
         Self::deposit_event(RawEvent::DidCreated(id, acc, vec![]));
     }
+
+    /// It returns the list of flatten identities of the given identity.
+    /// It runs recursively over all signing items.
+    pub fn flatten_identities(id: IdentityId) -> Vec<IdentityId> {
+        if <DidRecords>::contains_key(id) {
+            let identity = <DidRecords>::get(id);
+
+            identity
+                .signing_items
+                .into_iter()
+                .flat_map(|si| match si.signer {
+                    Signatory::Identity(sub_id) => Self::flatten_identities(sub_id),
```

Comment:
Not sure we should be recursing here.

For example if ALICE_IDENTITY has BOB_IDENTITY as a signer, and BOB_IDENTITY has CHARLES_IDENTITY as a signer, then:
`flatten_identities(ALICE_IDENTITY)` would return `[ALICE_IDENTITY, BOB_IDENTITY, CHARLES_IDENTITY]` but `CHARLES_IDENTITY` isn't directly related to `ALICE_IDENTITY` in the way you use this function for the governance RPC.